code/workspaces/test/test-results/10893_STS.md

```diff
@@ -30,879 +18,879 @@ This System Test Script has been produced during the project and any subsequent
+| 2 | Navigate to the datalabs homepage | User is presented with the login/sign up buttons |  |
+| 3 | Click on Sign Up | User is shown the login dialog. There is a sign up tab |  |
+| 4 | Select the sign up tab |There are two fields present for the user's email and password |  |
+| 5 | Enter a valid email and password | Email is displayed and the password is hidden |  |
+| 6 | Press sign up | The user is taken to a verification page where there are two buttons:Home and Verified. An email is sent to the user. |  |
+| 7 | Press the verified button | USer is taken to the log in screen |  |
+| 8 | Attempt to login without verifying the account  | User is taken back to the verification page  |  |
+| 9 | Verify the account via the email | User is taken to a new successful verification page  |  |
+| 10 | On the first verification page click on the verified button | User is taken to the login page |  |
+| 11 | Using the credentials used to sign up, sign into Datalabs | User can successfully sign in |  |
 ---
 
->>>###### **TC2.6.1.2 Configurable sign up**
+###### TC2.6.1.2 Configurable sign up
 
->>> | **JIRA ID(s):** | [NERCDL-867](https://jira.ceh.ac.uk/browse/NERCDL-867?jql=key%20in%20(NERCDL-867)) |OVERALL RESULT: | PASS |
->>> | ------------ | --------- | --------- | ------|
->>> | **Number** | **Step** | **Expected Result** | **Result** |
->>> | 1 | Ensure that the oidc configmap is set to its alternate values (see the linked Jira issue above) | The oidc configmap is set to its alternate values | PASS |
->>> | 2 | Ensure that the user is logged out of Datalabs | User is logged out | PASS |
->>> | 3 | Navigate to <https://testlab.test-datalabs.nerc.ac.uk/projects> | User is re-directed to the sign in page. There is a sign up button | PASS |
->>> | 4 | Click on the sign up button | User is re-directed to <https://testlab.test-datalabs.nerc.ac.uk/sign-up> and an email is created. This email is addressed to appsupport@ceh.ac.uk | PASS |
->>> | 5 | Click on the home button | User is re-directed to the Datalabs homepage | PASS |
->>> | 6 | Log in to Datalabs | User is logged in and re-directed to <https://testlab.test-datalabs.nerc.ac.uk/projects>  | PASS |
+| **JIRA ID(s):** | [NERCDL-867](https://jira.ceh.ac.uk/browse/NERCDL-867?jql=key%20in%20(NERCDL-867)) |OVERALL RESULT: |  |
+| ------------ | --------- | --------- | ------|
+| **Number** | **Step** | **Expected Result** | **Result** |
+| 1 | Ensure that the oidc configmap is set to its alternate values (see the linked Jira issue above) | The oidc configmap is set to its alternate values |  |
+| 2 | Ensure that the user is logged out of Datalabs | User is logged out |  |
+| 3 | Navigate to <https://testlab.test-datalabs.nerc.ac.uk/projects>| User is re-directed to the sign in page. There is a sign up button |  |
+| 4 | Click on the sign up button | User is re-directed to <https://testlab.test-datalabs.nerc.ac.uk/sign-upand> an email is created. This email is addressed to appsupport@ceh.ac.uk |  |
```

Comment:
looks like the `and` has made its way into the sign-up link

Reply:
Thanks, well spotted!